packages/jest/src/executors/jest/schema.json

```diff
@@ -51,6 +51,13 @@
       "x-completion-type": "file",
       "x-completion-glob": "jest.config@(.js|.ts)"
     },
+    "_": {
+      "type": "array",
+      "description": "The name of the file to test as a single positional argument. Equivalent to `--testFile`.",
+      "items": {
+        "type": "string"
+      }
+    },
```

Comment:
We don't need an extra option for this. We can specify a positional argument as the default source for any option (https://nx.dev/extending-nx/recipes/generator-options#default).

So, instead of adding this new option, you can edit the existing `testFile` to:

```diff
    "testFile": {
      "description": "The name of the file to test.",
      "type": "string",
+     "$default": {
+       "$source": "argv",
+       "index": 0
+     }
    },
```

With that in place, you get the desired behavior and don't need to handle any extra `_` option. The `testFile` option will get assigned the value of the first positional argument when the `--testFile` flag is not provided. The `--testFile` flag can still be used, in which case, it will override the positional argument if also provided.